sdk/key_vault/src/key.rs

```diff
@@ -0,0 +1,334 @@
+use std::fmt::{Debug, Display};
+
+use azure_core::TokenCredential;
+use chrono::serde::ts_seconds_option;
+use chrono::{DateTime, Utc};
+use getset::Getters;
+use serde::Deserialize;
+use serde_json::{Map, Value};
+use url::Url;
+
+use crate::client::API_VERSION;
+use crate::{KeyVaultClient, KeyVaultError};
+
+/// A KeyBundle consisting of a WebKey plus its attributes.
+#[derive(Debug, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct KeyBundle {
+    /// The key management attributes.
+    attributes: KeyAttributes,
+    /// The Json web key.
+    key: JsonWebKey,
+    /// True if the key's lifetime is managed by key vault. If this is a key backing a certificate, then managed will be true.
+    managed: Option<bool>,
+    /// Application specific metadata in the form of key-value pairs.
+    tags: Option<Map<String, Value>>,
```

Comment:
`tags` and `managed` should hang off `KeyProperties`, which mainly wrap `KeyAttributes`.